src/sysdolphin/baselib/initialize.c

```diff
@@ -264,7 +264,7 @@ static void HSD_ObjInit(void)
     HSD_ZListInitAllocData();
 }
 
-static char unused[] = "pix_fmt != GX_PF_RGB565_Z16";
+static u8 unused[] = "pix_fmt != GX_PF_RGB565_Z16";
```

Comment:
```suggestion
static char unused[] = "pix_fmt != GX_PF_RGB565_Z16";
```
this one really is a char array ;)

Reply:
:see_no_evil: